setup.sql

```diff
@@ -107,7 +109,8 @@ CREATE TABLE `CampaignNotePageDetail` (
   `updatedAt` datetime NOT NULL,
   `deletedAt` datetime DEFAULT NULL,
   PRIMARY KEY (`ID`),
-  KEY `detailInCampaign` (`Campaign_ID`,`PageDetail_ID`)
+  KEY `detailInCampaign` (`Campaign_ID`,`PageDetail_ID`),
+  KEY `notePartitionRange` (`partition_noteStart`,`partition_noteEnd`)
```

Comment:
Why do we need to keep track of this?  I would have thought the db kept track on its own.